networking/templates/vrrp-notify-script.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+TYPE=$1
+NAME=$2
+STATE=$3
+
+case $STATE in
+        "MASTER") systemctl start bird
+                  systemctl start haproxy
+                  exit 0
+                  ;;
+        "BACKUP") systemctl stop bird
+                  systemctl stop haproxy
+                  exit 0
+                  ;;
+        "FAULT")  systemctl stop bird
+                  systemctl stop haproxy
+                  exit 0
+                  ;;
```

Comment:
How are these different?

Reply:
I'm asking since you differentiated them. Not that big of a deal.

Reply:
> Thankfully, Keepalived also has the ability to run an arbitrary script to determine the health of a host. You can adjust the weight of the script, but I’m going to keep things simple for this example: a script that returns 0 will indicate success, while a script that returns anything else will indicate that the Keepalived instance should enter the fault state. 

Taken From https://www.redhat.com/sysadmin/advanced-keepalived. While fault states are not implemented yet, they might be useful, in the future, to make the VRRP instance enter the `FAULT`state if the internet fails and tries to failover to another router. In case of such a fault state, we have to stop Bird and HAProxy because these services can only run on one router node at a time.

Reply:
If that's the case, it should be explicitly stated in the file, imo. Other than that, perhaps making the `FAULT` state exit with a code of `1` could make more sense, but I'll leave that up to you.

Reply:
I think it should exit with status `0` because it should only fail with `1` if the notify script itself fails and not if it receives a `FAULT`. The [reference documentaton](https://docs.oracle.com/en/operating-systems/oracle-linux/6/admin/section_hxz_zdw_pr.html)  also implements it this way.

Reply:
But I've also added the NOTE for future improvements to the router redundancy